extensions/wikia/AdEngine/js/provider/gpt/helper.js

```diff
@@ -52,6 +58,12 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 			'INCONTENT_PLAYER'
 		];
 
+	if (fvLagger) {
+		fvLagger.addResponseListener(function () {
+			adEngineBridge.universalAdPackage.setUapId(fvAd.getLineItemId());
```

Comment:
We can pass lineItemId/ima ad as an argument in `adResponseListener` and don't expose new method `getLineItemId`. What do you think?

Reply:
It will be executed on each `adRequest` event

Reply:
Sounds good & clean

Reply:
What about consecutive pvs on mobile-wiki?